src/api/app/policies/groups_user_policy.rb

```diff
@@ -0,0 +1,6 @@
+class GroupsUserPolicy < ApplicationPolicy
```

Comment:
Do we really need to create a GroupsUserPolicy? I wonder if we could modify the `destroy?` method of GroupPolicy, here: https://github.com/openSUSE/open-build-service/blob/e96f6256a1de2ea10ccf58ffaa3f52173e1e8e32/src/api/app/policies/group_policy.rb#L15-L17

Reply:
`GroupPolicy#destroy?` is about deleting groups. `GroupsUserPolicy#destroy?` is about removing users from a group. Those are two different actions, so this is why we have this new `GroupsUserPolicy` now.

Reply:
I see. Thanks for the explanation.